pkg/sql/distsql_plan_bulk.go

```diff
@@ -86,12 +80,104 @@ func (dsp *DistSQLPlanner) setupAllNodesPlanningTenant(
 	for i, pod := range pods {
 		sqlInstanceIDs[i] = pod.InstanceID
 	}
-	// Shuffle node order so that multiple IMPORTs done in parallel will not
-	// identically schedule CSV reading. For example, if there are 3 nodes and 4
-	// files, the first node will get 2 files while the other nodes will each get 1
-	// file. Shuffling will make that first node random instead of always the same.
-	rand.Shuffle(len(sqlInstanceIDs), func(i, j int) {
-		sqlInstanceIDs[i], sqlInstanceIDs[j] = sqlInstanceIDs[j], sqlInstanceIDs[i]
-	})
 	return planCtx, sqlInstanceIDs, nil
 }
+
+// PhysicalPlanMaker describes a function that makes a physical plan.
+type PhysicalPlanMaker func(context.Context, *DistSQLPlanner) (*PhysicalPlan, *PlanningCtx, error)
+
+// calculatePlanGrowth returns the number of new or updated sql instances in one
+// physical plan relative to an old one, as well as that number as a fraction of
+// the number in the old one.
+func calculatePlanGrowth(before, after *PhysicalPlan) (int, float64) {
```

Comment:
I wonder if it makes sense to move this to standalone file. With test :)

Reply:
I think this is the right file -- this file has non-query distsql planning helpers, which is what this is. 

Added test.

Reply:
ack